pkg/asset/agent/manifests/agentclusterinstall.go

```diff
@@ -276,6 +277,8 @@ func (a *AgentClusterInstall) Load(f asset.FileFetcher) (bool, error) {
 	switch string(agentClusterInstall.Spec.PlatformType) {
 	case baremetal.Name:
 		agentClusterInstall.Spec.PlatformType = hiveext.BareMetalPlatformType
+	case external.Name:
+		agentClusterInstall.Spec.PlatformType = hiveext.ExternalPlatformType
```

Comment:
Based on the latest version of https://github.com/openshift/assisted-service/pull/5438 we'll need to set external.platformName here as well.